plat/hisilicon/hikey/hikey_bl1_setup.c

```diff
@@ -42,8 +42,11 @@ meminfo_t *bl1_plat_sec_mem_layout(void)
  */
 void bl1_early_platform_setup(void)
 {
+	static console_pl011_t console;
```

Comment:
I'd rather move this out of the function. It should be clear that this isn't in the stack. Same for the rest of the files.

Reply:
Sure. FYI, this pattern appears twice in the current code (with `console_16550_t` in `rockchip` and `ti/k3`).

Reply:
Ok, that's not ideal...

In any case, the CI is already reporting build failures for your modified platforms. You need to modify the ``plat_crash_console_xxx`` functions.

Reply:
How do I see the build failures?
```
jenkins.oss.arm.com’s server IP address could not be found.
```

Reply:
Yeah, the CI isn't public (but hopefully that will change over the next few months).  All you need to do to see the errors is:
```
CROSS_COMPILE=aarch64-linux-gnu- make PLAT=hikey
CROSS_COMPILE=aarch64-linux-gnu- make PLAT=hikey960
```
That should fail to compile.